### ISSUE TITLE
Integrative Deep Models for Alternative Splicing

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#238]
https://doi.org/10.1101/104869

> Advancements in sequencing technologies have highlighted the role of alternative splicing (AS) in increasing transcriptome complexity. This role of AS, combined with the relation of aberrant splicing to malignant states, motivated two streams of research, experimental and computational. The first involves a myriad of techniques such as RNA-Seq and CLIP-Seq to identify splicing regulators and their putative targets. The second involves probabilistic models, also known as splicing codes, which infer regulatory mechanisms and predict splicing outcome directly from genomic sequence. To date, these models have utilized only expression data. In this work we address two related challenges: Can we improve on previous models for AS outcome prediction and can we integrate additional sources of data to improve predictions for AS regulatory factors. We perform a detailed comparison of two previous modeling approaches, Bayesian and Deep Neural networks, dissecting the confounding effects of datasets and target functions. We then develop a new target function for AS prediction and show that it significantly improves model accuracy. Next, we develop a modeling framework to incorporate CLIP-Seq, knockdown and over-expression experiments, which are inherently noisy and suffer from missing values. Using several datasets involving key splice factors in mouse brain, muscle and heart we demonstrate both the prediction improvements and biological insights offered by our new models. Overall, the framework we propose offers a scalable integrative solution to improve splicing code modeling as vast amounts of relevant genomic data become available. Availability: code and data will be available on Github following publication.

**Comment by gwaygenomics** [comment]
Clear and comprehensive analysis making several comparisons against previous models. The authors nicely demonstrate how a deep neural network integrating diverse data types (such as [CLIP-seq](https://en.wikipedia.org/wiki/CLIP) and gene knockout) with genomic sequence features to predict alternative splice sites improves state of the art prediction. They compare this model to Bayesian neural networks and a previous model discussed in #4. Additionally, because of increasing amounts of available data and algorithm capacity, the authors are also able to develop a target function to model exon inclusion directly instead of categorizing exon inclusion as low, mid, or high.

# Biological Aspects

* Predict alternative splicing events from sequence features
  * Also determine if integrating CLIP-seq and knockout/knockdown data improve model
* Compare sequence feature based models to "ground truth" RNAseq inferred exon inclusion

# Computational Aspects

* Thorough comparison of four models
  * Deep Bayesian Network
  * Deep Neural Network implemented in #4 
  * Deep Neural Network with CLIP-seq integration
  * Deep Neural Network with Knockout/Knockdown integration
* Also very clear training processes and hyperparameter selection rationale

# Why we should include it in our review

Alternative splicing is pervasive and has already been shown to be important in disease biology and to understand molecular function. Deep learning has already shown promise to improve prediction, which could lead directly to more discoveries.
